extensions/amp-viewer-assistance/0.1/test/validator-amp-viewer-assistance.out

```diff
@@ -0,0 +1,39 @@
+PASS
```

Comment:
This file already exists as [validator-actions-amp-viewer-assistance.out](https://github.com/ampproject/amphtml/blob/master/extensions/amp-viewer-assistance/0.1/test/validator-actions-amp-viewer-assistance.out)

Reply:
Removed.